cache-config/t3c-check-reload/README.md

```diff
@@ -58,25 +59,14 @@ Possible return values are:
 
   '' - no reload or restart is necessary.
 
-# OPTIONS
-
--c, --changed-config-paths=value
+# JSON Format
 
-    comma-delimited list of the full paths of all files changed
-    by t3c
+    {"changed_files":"<list of files>","installed_plugins":"<list of plugins>"}
+# OPTIONS
 -h, --help
 
     Print usage information and exit
 
--m, --run-mode=value
-
-     [badass | report | revalidate | syncds] run mode, default is
-     'report' [report]
-
--p, --plugin-packages-installed=value
-
-    comma-delimited list of ATS plugin packages which were
-    installed by t3c
```

Comment:
I can merge when conflicts are fixed - but quick question: is this gonna cause upgrade issues with like cron jobs or something? I don't see anything that would change cron jobs in the spec file or anything.

Reply:
Conflicts have been fixed it was just in the changelog. since t3c-check-reload is called inside of t3c-apply the cron job should not have to change. I remove --run-mode from the README.md since it was never used. The other 2 flags are replace by json input.

Reply:
Alright, shame we don't support calling `t3c check-reload` by itself but that's nothing new - atstccfg was the same way, so if anyone's doing that it's on them, I suppose.